llpc/CMakeLists.txt

```diff
@@ -131,7 +131,7 @@ if(ICD_BUILD_LLPC)
     endif()
 
     # Always link statically against libLLVMlgc
-    llvm_map_components_to_libnames(extra_llvm_libs lgc)
+    llvm_map_components_to_libnames(extra_llvm_libs lgc LgcCps)
```

Comment:
Hi Ruiling. You only needed to add this dependency on LgcCps because of the code you added to modify the data layout. Could you instead do that data layout modification when you get to a pass in LgcRt that lowers to LgcCps ops?

Reply:
I think that should be possible in the future when we get the passes that lowers to LgcCps ready. For now, let's do it here to to make cps development easy.